environment.yml

```diff
@@ -1,36 +1,41 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-sarek-3.0dev
+name: nf-core-sarek-2.7
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
-  - conda-forge::markdown=3.1.1
-  - conda-forge::pymdown-extensions=6.0
-  - conda-forge::pygments=2.5.2
-  - bioconda::ascat=2.5.2
-  - bioconda::bcftools=1.9
-  - bioconda::bwa=0.7.17
-  - bioconda::cancerit-allelecount=4.0.2
-  - bioconda::cnvkit=0.9.6
-  - bioconda::control-freec=11.5
-  - bioconda::ensembl-vep=99.2
-  - bioconda::fastqc=0.11.9
-  - bioconda::freebayes=1.3.2
-  - bioconda::gatk4-spark=4.1.7.0
-  - bioconda::genesplicer=1.0
-  - bioconda::htslib=1.9
-  - bioconda::manta=1.6.0
-  - bioconda::msisensor=0.5
-  - bioconda::multiqc=1.8
-  - bioconda::qualimap=2.2.2d
-  - bioconda::samtools=1.9
-  - bioconda::snpeff=4.3.1t
-  - bioconda::strelka=2.9.10
-  - bioconda::tiddit=2.7.1
-  - bioconda::trim-galore=0.6.5
-  - bioconda::vcfanno=0.3.2
-  - bioconda::vcftools=0.1.16
-  - conda-forge::pigz=2.3.4 
-  - conda-forge::r-ggplot2=3.3.0
+  - conda-forge::llvm-openmp=8.0.1=hc9558a2_0
+  - conda-forge::openmp=8.0.1=0
+  - conda-forge::markdown=3.1.1=py_0
+  - conda-forge::pymdown-extensions=6.0=py_0
+  - conda-forge::pygments=2.5.2=py_0
+  - bioconda::ascat=2.5.2=r40_2
+  - bioconda::bcftools=1.9=ha228f0b_4
```

Comment:
I'm just not sure here regarding the conda environment, do we add now also the exact build?

Reply:
cf: #323
No idea why, but with that I can actually solve the `env`.
Without, it hangs for more than 12 hours.
Anyway, we're ditching this conda env for the 3.0 release, so it works, it's reproducible, I'm happy enough with it :-D